### PR TITLE
[xlnt] Add save fuzz target

### PR DESCRIPTION
**Pull request by Avgor46**
Maybe we should just keep only one fuzz target...

### CODE REVIEW

**Review by kobrineli — CHANGES_REQUESTED** [review]
Update licenses in dockerfile and build. Add license to new target code.

**Review by kobrineli — APPROVED** [review]

File: projects/xlnt/build.sh [hunk 1]
```diff
@@ -28,9 +28,11 @@ CMAKE_BUILD_PARALLEL_LEVEL=$(nproc) cmake --build .
 CXX="clang++"
 CXXFLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero"
 
-$CXX $CXXFLAGS -I/xlnt/include -I/xlnt/third-party/libstudxml -O2 -o load_fuzzer.o -c ../load_fuzzer.cc
+$CXX $CXXFLAGS -I/xlnt/include -I/xlnt/third-party/libstudxml -O2 -o load_fuzzer.o -c ../load.cc
```
Comment (PaDarochek) [inline projects/xlnt/build.sh:31]: Maybe we can compile and link within same command?

File: projects/xlnt/save-afl++.toml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+################################################################################
+
+[sydr]
+args = "-l debug -j2"
+target = "/save_sydr @@"
+jobs = 2
+
+[aflplusplus]
+args = "-t 1000 -i /save_corpus"
+target = "/save_afl"
```
Comment (PaDarochek) [inline projects/xlnt/save-afl++.toml:24]: Add `@@`

**Review by PaDarochek — CHANGES_REQUESTED** [review]

File: projects/xlnt/build_save_corpus.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+cp -r /corpus /save_corpus
```
Comment (apach301) [inline projects/xlnt/build_save_corpus.sh:17]: I think that 2-line bash command can be written directly in Dockerfile

File: projects/xlnt/build.sh (hunk 1 above)
Comment (Avgor46) [reply projects/xlnt/build.sh]: done

File: projects/xlnt/build_save_corpus.sh (hunk 3 above)
Comment (Avgor46) [reply projects/xlnt/build_save_corpus.sh]: done

**Review by apach301 — APPROVED** [review]

**Review by PaDarochek — APPROVED** [review]

**Pull request merged**